test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -1297,9 +1246,163 @@ pub fn test_delete_access_key_with_bob_refund(node: impl Node) {
     let transaction_result = node_user.get_transaction_result(&transaction_result.receipts[0]);
     assert_eq!(transaction_result.status, TransactionStatus::Completed);
     assert!(transaction_result.receipts.is_empty());
-    let new_root1 = node_user.get_state_root();
-    assert_ne!(new_root1, new_root);
+    let new_root = node_user.get_state_root();
+    assert_ne!(new_root, root);
 
     let bob_account_result = node_user.view_account(&bob_account()).unwrap();
     assert_eq!(bob_account_result.amount, bobs_initial_balance + 10);
 }
+
+pub fn test_access_key_smart_contract(node: impl Node) {
+    let access_key = AccessKey {
+        amount: 0,
+        balance_owner: None,
+        contract_id: Some(bob_account()),
+        method_name: None,
```

Comment:
If `method_name` is None, does it mean it can access any method in the contract?

Reply:
Yes